deploy/operator.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: observatorium-operator
+  labels:
+    control-plane: observatorium-operator
```

Comment:
let's add the kubernetes app labels here like we do everywhere else

Reply:
Done